tensorrt_llm/_torch/visual_gen/models/qwen_image/transformer_qwen_image.py

```
@@ -38,6 +38,84 @@
+        # pre_quant_scale ([in_features]) may already be loaded by the base NVFP4
+        # method on newer releases; load it here too for robustness.
+        if getattr(module, "pre_quant_scale", None) is None and "pre_quant_scale" in w:
+            module.pre_quant_scale = nn.Parameter(
+                w["pre_quant_scale"].to(device), requires_grad=False)
+        if "svdquant_lora_a" in w:
+            module.svdquant_lora_a = nn.Parameter(
+                w["svdquant_lora_a"].to(device), requires_grad=False)
+            module.svdquant_lora_b = nn.Parameter(
+                w["svdquant_lora_b"].to(device), requires_grad=False)
+
+    def apply(self, module, input, bias):
```

Ruff (0.15.18)

[error] Function argument `input` is shadowing a Python builtin

(A002)